test/unit/test_client.py

```diff
@@ -1,24 +1,52 @@
+import warnings
+
 import pytest
 
 import ipfshttpclient
 
 
 def test_assert_version():
-	# Minimum required version
-	ipfshttpclient.assert_version("0.1.0", "0.1.0", "0.2.0", ["0.1.2"])
-	
+	with warnings.catch_warnings(record=True) as w:
+		# Cause all warnings to always be triggered.
+		warnings.simplefilter("always")
+
+		# Minimum required version
+		ipfshttpclient.assert_version("0.1.0", "0.1.0", "0.2.0", ["0.1.2"])
+
+		assert len(w) == 0
```

Comment:
This first assertion can apparently also be shortend using `pytest.warns`: https://stackoverflow.com/a/45671804/277882